tests/cli/test_orion.py

```diff
@@ -56,3 +56,38 @@ def test_start_with_keep_alive_from_setting(mock_run_process: AsyncMock):
     command: List[str] = mock_run_process.call_args[1]["command"]
     assert "--timeout-keep-alive" in command
     assert command[command.index("--timeout-keep-alive") + 1] == "100"
+
+
+def test_start_from_cli_with_keep_alive(mock_run_process: AsyncMock):
+    invoke_and_assert(
+        ["orion", "start", "--keep-alive-timeout", "100"],
+        expected_output_contains=[
+            "prefect config set PREFECT_API_URL=http://127.0.0.1:4200/api",
+            "View the API reference documentation at http://127.0.0.1:4200/docs",
+        ],
+    )
+
+    mock_run_process.assert_awaited_once()
+    command: List[str] = mock_run_process.call_args[1]["command"]
+    assert "--timeout-keep-alive" in command
+    assert command[command.index("--timeout-keep-alive") + 1] == "100"
+
+
+def test_start_from_cli_with_invalid_keep_alive(mock_run_process: AsyncMock):
+    invoke_and_assert(
+        ["orion", "start", "--timeout-keep-alive", "45"],
+        expected_code=2,
+        expected_output_contains=[
+            "No such option: --timeout-keep-alive Did you mean --keep-alive-timeout?",
+        ],
+    )
+
+
+def test_start_help_from_cli_has_keep_alive(mock_run_process: AsyncMock):
+    invoke_and_assert(
+        ["orion", "start", "--help"],
+        expected_output_contains=[
+            "--keep-alive-timeout",
+            "PREFECT_ORION_API_KEEPA",
+        ],
```

Comment:
Is there a clearer assertion we can do here? This doesn't seem like it will prevent the help menu from becoming malformed in the future. We could probably skip this test entirely if it's hard to make an assertion about the help options — that kind of thing is covered by the `typer` tests.

Reply:
The intent is to make sure that this setting is not removed accidentally and tbh I was not thinking about the malformed help menu, with the assumption that is covered elsewhere.

Reply:
Ah accidental removal of the setting should be covered by the other tests (like the one I wrote initially).

Reply:
Yes of ocurse I can see it now that you mention it. I have removed this test